examples/convert_mri_and_trans.py

```diff
@@ -123,12 +123,21 @@
 t1w_bids_path = \
     BIDSPath(subject=sub, session=ses, root=output_path, suffix='T1w')
 
+# use ``trans`` to transform landmarks from the ``raw`` file to
+# the voxel space of the image
+landmarks = get_landmarks(
+    t1_mgh_fname,  # path to the MRI scan
```

Comment:
calling this t1_mgh_fname suggests get_landmarks needs mgh data in the T1. I think it's misleading. The T1 MGZ will be taking from subject / subjects_dir